manifests/redhat.pp

```diff
@@ -36,27 +36,27 @@
 
     $public_key_local = '/etc/pki/rpm-gpg/DATADOG_RPM_KEY.public'
 
-    file { 'DATADOG_RPM_KEY_E09422B3.public':
+    file { 'DATADOG_RPM_KEY_20200908.public':
         owner  => root,
         group  => root,
         mode   => '0600',
         path   => $public_key_local,
-        source => 'https://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public'
+        source => 'https://yum.datadoghq.com/DATADOG_RPM_KEY_20200908.public'
     }
 
     exec { 'install-gpg-key':
         command => "/bin/rpm --import ${public_key_local}",
-        onlyif  => "/usr/bin/gpg --dry-run --quiet --with-fingerprint -n ${public_key_local} | grep 'A4C0 B90D 7443 CF6E 4E8A  A341 F106 8E14 E094 22B3' || gpg --dry-run --import --import-options import-show ${public_key_local} | grep 'A4C0B90D7443CF6E4E8AA341F1068E14E09422B3'",
-        unless  => '/bin/rpm -q gpg-pubkey-e09422b3',
-        require => File['DATADOG_RPM_KEY_E09422B3.public'],
+        onlyif  => "/usr/bin/gpg --dry-run --quiet --with-fingerprint -n ${public_key_local} | grep 'C655 9B69 0CA8 82F0 23BD  F3F6 3F4D 1729 FD4B F915' || gpg --dry-run --import --import-options import-show ${public_key_local} | grep 'C6559B690CA882F023BDF3F63F4D1729FD4BF915'",
+        unless  => '/bin/rpm -q gpg-pubkey-fd4bf915',
+        require => File['DATADOG_RPM_KEY_20200908.public'],
     }
```

Comment:
I think this means that if the Agent 5 or 6 repo is used, the `E09422B3` key is not installed.

For Agent 5, it's not an immediate problem (Agent 5 packages are signed with the old key), but this probably breaks Agent 6 installs, as they'll have the old key (`DATADOG_RPM_KEY.public`) and the future key (`DATADOG_RPM_KEY_20200908.public`) installed, but not the current one (`DATADOG_RPM_KEY_E09422B3.public`).

This can be solved by either:
- updating the `$gpgkeys` above (see other comment)
- adding the section that was removed
- doing both (but it's probably not needed?)

I even think that if you add all the correct keys to `$gpgkeys`, you won't need this section at all, given that `yum` will import the keys in `$gpgkeys` when installing packages from the repo.

Reply:
> I even think that if you add all the correct keys to $gpgkeys, you won't need this section at all, given that yum will import the keys in $gpgkeys when installing packages from the repo.

That was my impression too, I am not super sure why that code is there, cc @albertvaka in case he can shed some light into this

Reply:
I removed the section altogether and it worked fine for installing Agent 6 and 7 (5 failed but at a different unrelated step)